src/sas/sascalc/dataloader/readers/cansas_reader.py

```diff
@@ -1056,6 +1058,8 @@ def _write_data(self, datainfo, entry_node):
             sesans = self.create_element("Sesans")
             sesans.text = str(datainfo.isSesans)
             node.append(sesans)
+            self.write_node(node, "yacceptance", datainfo.sample.yacceptance[0],
```

Comment:
There is currently no way to load in yacceptance into sasview. When saving a project or sesans data set as XML, the default value of None will throw an error here. Setting the default of yacceptance (zacceptance should also probably be changed) to a tuple of (0,"") might be the safer way to go.